uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -50,35 +50,14 @@ $VERSION = 0.2;
 @EXPORT = qw();
 @EXPORT_OK = qw(identify_scan in_range get_headers get_info get_ids get_objective identify_scan_db scan_type_text_to_id scan_type_id_to_text register_db get_header_hash get_scanner_id get_psc compute_hash is_unique_hash make_pics select_volume);
 
-=pod
-B<getSubjectIDs( C<$patientName>, C<$scannerID>, C<$dbhr> )>
-Determines the cand id and visit label for the subject based on patient name and (for calibration data) scannerid.
-Returns: a reference to a hash containing elements including 'CandID', 'visitLabel' and 'visitNo', or, in the case of failure, undef
-=cut
-sub getSubjectIDs {
-    my ($patientName, $scannerID, $dbhr) = @_;
-    my %subjectID;
-# calibration data (PHANTOM_site_date | LIVING_PHANTOM_site_date | *test*)
-    if ($patientName =~ /PHA/i or $patientName =~ /TEST/i) {
-	$subjectID{'CandID'} = my_trim(getScannerCandID($scannerID, $dbhr));
-	$subjectID{'visitLabel'} = my_trim($patientName);
-# subject data       	
-# old versions of this
-# qnts /([A-Z-]{3,4}\s+\d+)_(\d+)_([^_ ]+)/) or nihpd =~ /(\w{3}\d+)_(\d+)_([^_ ]+)/)
-    } elsif ($patientName =~ /([^_]+)_(\d+)_([^_ ]+)/) {
-	$subjectID{'PSCID'} = my_trim($1);
-	$subjectID{'CandID'} = my_trim($2);
-	$subjectID{'visitLabel'} = my_trim($3);
-	if(!subjectIDIsValid($subjectID{'CandID'}, $subjectID{'PSCID'}, $subjectID{'visitLabel'}, $dbhr)) {
```

Comment:
@spaiva, @cmadjar:
We just need to be careful that this duplicate getSubjectID() function that got removed did not also remove some additional functionalities... for example subjectIDisValid() which itself also calls other functions... this is not used in the getSubjectID() in the prod file. So we need to investigate this further.